unreleased/remove_sw_receiver_warning_msg.yaml

```diff
@@ -0,0 +1,13 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: skywalkingreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: remove receiver warning message
+
+# One or more tracking issues related to the change
+issues: [13550]
+
+subtext:
```

Comment:
Such changes don't need a changelog entry. `Skip Changelog` github label should be applied instead